### PR TITLE
2269 remove exploit telem

### PR DESCRIPTION
**Pull request by cakekoa**
# What does this PR do?

Fixes part of #2269.

Removes ExploitTelem and the code that processes it.

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] ~~Was the CHANGELOG.md updated to reflect the changes?~~
* [ ] ~~Was the documentation framework updated to reflect the changes?~~
* [x] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] ~~Added relevant unit tests?~~
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by running in the zoo environment
* [x] If applicable, add screenshots or log transcripts of the feature working
    > Screen Shot 2022-10-13 at 4 49 59 PM
    > Screen Shot 2022-10-13 at 4 09 52 PM
    > Screen Shot 2022-10-13 at 4 08 44 PM
    > Screen Shot 2022-10-13 at 4 09 23 PM

### CODE REVIEW

**Review by VakarisZ — CHANGES_REQUESTED** [review]
Exploit telemetries are still being used in `monkey/monkey_island/cc/resources/monkey.py:49`. If that endpoint is no longer relevant, we should remove it here

**Comment by VakarisZ** [comment]
`test_monkey_exploitation.py` also using it

**Comment by VakarisZ** [comment]
There are a bunch of ATT&CK technique reports also querying this type of telemetry

**Comment by cakekoa** [comment]
Let me know if you think this is appropriate:
  - I will remove the code pertaining to exploits in the Monkey resource. It appears that the database that it updates is used in a number of places still, and we can reconsider removing it when we rework the map.
  - I have removed `test_monkey_exploitation.py` as well as the method that it tested.
  - As we discussed, since ATT&CK report is being removed, I think we can just skip any changes here for now.

**Review by shreyamalviya — APPROVED** [review]

**Comment by VakarisZ** [comment]
> Let me know if you think this is appropriate:
> 
> * I will remove the code pertaining to exploits in the Monkey resource. It appears that the database that it updates is used in a number of places still, and we can reconsider removing it when we rework the map.
> * I have removed `test_monkey_exploitation.py` as well as the method that it tested.
> * As we discussed, since ATT&CK report is being removed, I think we can just skip any changes here for now.

1. OK
2. OK
3. Make sure that other functionality is not broken by this, for example, there should be no 500 errors when I go to the report/map etc.

File: monkey/monkey_island/cc/resources/monkey.py [hunk 1]
```diff
@@ -42,43 +42,7 @@ def post(self, **kw):
...
-                        }
-                    )
-                ]
-
-                if 1 == len(exploit_telem):
-                    parent_to_add = (
-                        exploit_telem[0].get("monkey_guid"),
-                        exploit_telem[0].get("data").get("exploiter"),
-                    )
+                parent_to_add = (parent, None)
```
[the remark this answers is not in the document]
Comment (VakarisZ) [reply monkey/monkey_island/cc/resources/monkey.py:45]: I don't think we can do it yet without breaking the report

**Pull request merged**